index.html

```diff
@@ -2,9 +2,9 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" type="image/svg+xml" href="/typefaceoff_logo.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React + TS</title>
+    <title>TypeFaceOff</title>
```

Comment:
Seeing as we’ve now decided on an “official” capitalisation, might get you to update this.

Sorry about the busywork 😅